src/ontology/uberon-edit.obo

```diff
@@ -223782,6 +223781,16 @@ relationship: part_of UBERON:0000992 ! ovary
 property_value: dcterms-date 2022-04-19T11:08:10Z xsd:dateTime
 property_value: seeAlso https://github.com/obophenotype/uberon/issues/476 xsd:anyURI
 
+[Term]
+id: UBERON:8450002
+name: excretory system
+def: "An anatomical system that eliminates waste products that arise as a result of metabolic activity." [GO:0007588]
+intersection_of: UBERON:0000467 ! anatomical system
+intersection_of: capable_of GO:0046903 ! secretion
+relationship: existence_ends_during UBERON:0000066 ! fully formed stage
```

Comment:
@gouttegd, I'm unsure about this axiom. Why does existence of the 'excretory system' end at a fully formed stage? It looks like this was pointed out before in other instances, but never addressed: https://github.com/geneontology/go-ontology/issues/14501

Reply:
Good question. I am not sure what those `existence_ends_*` axioms are supposed to model…

Most other anatomical systems have a similar axiom (including the old `renal system` that is being redefined in this PR). And on top of that, `anatomical structure` has a `existence_ends_during_or_before some 'death stage'`.

So as a result, we have anatomical systems whose existence is either said to
* “end during fully formed stage” (if there’s an explicit axiom on the anatomical system itself)
* “end during or before death stage” (without an explicit axiom, using the one inherited from `anatomical structure`).

What is the difference between the two supposed to mean? Frankly I don’t know. For now, I applied the “existence ends during fully formed stage” axiom to `excretory system` solely on the ground that the axiom was already applied to `renal system`. I don’t mind removing it and maybe leaving that to a later issue where we would review all uses of `existence_ends_*` axioms.

Reply:
> I don’t mind removing it and maybe leaving that to a later issue where we would review all uses of existence_ends_* axioms.

Sounds reasonable.